styles/general.css

```diff
@@ -196,3 +196,41 @@ footer {
 .medium-horizontal-padding {
  padding: 0 3rem;
 }
+
+
+
+/* Sidebar styling */
+.sidebar {
+    background-color: #F5F0D7; /* Main background for sidebar */
+    padding: var(--base-padding);
+    border-radius: var(--border-radius);
+}
+
+/* Sidebar Button Styling */
+.sidebar-button {
+    background-color: transparent;
+    border: none;
+    padding: var(--small-padding) var(--base-padding);
+    font-size: var(--base-font);
+    color: var(--textcolor-main);
+    cursor: pointer;
+    transition: all 0.3s ease; /* Smooth transition for hover effect */
```

Comment:
can take away tranisition here since it will look goofy when you highlight them fast.

Reply:
can also add border-radius: var(--border-radius)". We use that on other similiar buttons